### PR TITLE
Amishra hometown

### PR DESCRIPTION
**Pull request by arunmishra305**

### CODE REVIEW

**Comment by arunmishra305** [comment]
@niroulamohan : Please commit the changes!!

File: github-amishra.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## this is a github training
+-trying to learn some cool stuff
+-github seems very interesting
```
Comment (githubteacher) [inline github-amishra.md:3]: I am excited you find it interesting! It is a lot of fun to use :tada:

**Comment by iamjeffdunn** [comment]
Sup?

**Comment by niroulamohan** [comment]
are you excited?

**Comment by arunmishra305** [comment]
@iamjeffdunn : yo Monday!!

**Comment by arunmishra305** [comment]
@niroulamohan : very excited ...are you ?

File: github-amishra.md (hunk 1 above)
Comment (arunmishra305) [inline github-amishra.md:3]: ek;rtn

**Pull request merged**